### DISCUSSION TITLE
Welcome to vercel-storage Discussions!

### DISCUSSION THREAD

**Discussion by misaelnieto** [misaelnieto/vercel-storage#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fezzrootech** [comment]
Hello. I am an avid user of Vercel and was searching Python libs or SDK for Python and came across this.

Any chance the same thing can be used with Django framework? I would like to know if there is any way for the same.

Thanks in advance and great job making this library.
:)

**Comment by misaelnieto** [comment]
Hi, I wrote this with django in mind. Someone (maybe me?) will need to
write the django storage adapter using this library. I may be able to do it
in the future, but I just wanted to wait and see if this little library
attracts interest.

El sáb, 30 mar 2024 a la(s) 7:28 p.m., fezzrootech ***@***.***)
escribió:

> Hello. I am an avid user of Vercel and was searching Python libs or SDK
> for Python and came across this.
>
> Any chance the same thing can be used with Django framework? I would like
> to know if there is any way for the same.
>
> Thanks in advance and great job making this library.
> :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/misaelnieto/vercel-storage/discussions/2#discussioncomment-8962555>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABD3U7XHIA36AEA744XWBLY25YDNAVCNFSM6AAAAABECEOIUGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSNRSGU2TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Noe Nieto
M: ***@***.***
W: http://noenieto.com
T:  @misaelnieto_a <https://twitter.com/misaelnieto_a>
Li: Perfil en LinkedIn <https://www.linkedin.com/in/noe-nieto-13529524/>
